MarvelApp/MarvelAppTests/Tests/Network/NetworkServiceTests.swift

```diff
@@ -0,0 +1,23 @@
+@testable import MarvelApp
+
+import XCTest
+
+final class NetworkServiceTests: XCTestCase {
+    
+    private let cryptoSpy = MarvelCryptoSpy()
+    private let authSpy = AuthenticationURLProviderSpy()
+    
+    private lazy var sut = NetworkService(
+        crypto: cryptoSpy,
+        auth: authSpy
+    )
+    
+    func test_setNewCharacters_shouldReturnCorrectURL() {
```

Comment:
This test is simple like this because the responsibility for checking the URL parameters, base URL and methods is in the respective request model tests (SetNewCharacterRequestTests)